ccl_chromium_indexeddb.py

```diff
@@ -578,9 +578,15 @@ def read_record_precursor(
 
             if info is not None:
                 data_path = pathlib.Path(str(db_id), f"{info.blob_number >> 8:02x}", f"{info.blob_number:x}")
+                try:
+                    blob = self.get_blob(db_id, store_id, key.raw_key, externally_serialized_blob_index, data_path).read()
```

Comment:
some blobs were missing. i could not figure out cause.

i recently did update to a newer version of chrome, & open the session with that version. but previous to that, i had been running a very old version of chrome & had not used this session actively in quite a while. in case that helps explain why maybe some of these files had gone missing?

Reply:
The try/catch here seems sensible.